src/components/ReceiveMyAddress.js

```diff
@@ -47,7 +49,7 @@ export default function ReceiveMyAddress() {
       padding: 16,
       borderBottomWidth: 1.5,
       borderTopWidth: 1.5,
-      borderColor: '#e5e5ea',
+      borderColor: COLORS.borderColor,
```

Comment:
Change from `#e5e5ea` to `#eeeeee`

Reply:
Why?

Reply:
As discussed, this is close enough to the current border color to compose the app theme.